internal/mode/static/nginx/runtime/manager_test.go

```diff
@@ -21,8 +23,110 @@ var _ = Describe("NGINX Runtime Manager", func() {
 		mgr = NewManagerImpl(&ngxclient.NginxClient{}, nil, logr.New(GinkgoLogr.GetSink()))
 		Expect(mgr.IsPlus()).To(BeTrue())
 	})
+
+	Describe("ManagerImpl", Ordered, func() {
```

Comment:
This describe block is unnecessary, these tests can live under the parent describe block, like the existing test.